.goreleaser.yml

```diff
@@ -0,0 +1,31 @@
+before:
+  hooks:
+    - go mod download
+builds:
+- id: datadog-operator
```

Comment:
I think we don't need to build the `datadog-operator` with goreleaser since what we deliver is the Docker image.